src/components/tabs/Tabs.js

```diff
@@ -1,9 +1,27 @@
+import PropTypes from 'prop-types';
 import { TabNav } from './TabNav';
 
 export const Tab = () => null;
 
 export class Tabs extends Component {
-  state = { id: 0 }
+  // eslint-disable-next-line react/destructuring-assignment
+  state = { id: this.props.selectedIndex } // <-- please tell me, how I can this destructuring???
```

Comment:
- you should ask eslint authors)
- actually, you have `getDerivedStateFromProps` which is set state on first rendering